ProcessQueue/__init__.py

```diff
@@ -159,320 +190,343 @@ def main(msg: func.QueueMessage):
     else:
         data_dictionary = None
 
-    wb = openpyxl.load_workbook(scanreport, data_only=True)
+    wb = openpyxl.load_workbook(scanreport_bytes, data_only=True, keep_links=False)
 
     # Get the first sheet 'Field Overview',
     # to populate ScanReportTable & ScanReportField models
-    ws = wb.worksheets[0]
+    fo_ws = wb.worksheets[0]
 
+    # Get all the table names in the order they appear in the Field Overview page
     table_names = []
-    # Skip header with min_row=2
-    for i, row_cell in enumerate(ws.iter_rows(min_row=2), start=2):
-        # If the value in the first column (i.e. the Table Col) is not blank;
-        # Save the table name as a new entry in the model ScanReportTable
-        # Checks for blank b/c White Rabbit seperates tables with blank row
-        for cell in row_cell:
-            if cell.value:
-                name = ws.cell(row=i, column=1).value
-                if name not in table_names:
-                    table_names.append(name)
+    # Skip header row
+    for cell in fo_ws['A'][1:]:

```

Comment:
hmm not sure why this `'A'` is hardcoded?  Just making sure it wasn't from testing.

Reply:
`'A'` is selecting the first column to grab all the table names.